gr-uhd/lib/usrp_source_impl.h

```diff
@@ -11,6 +11,8 @@
 #include "usrp_block_impl.h"
 #include <gnuradio/uhd/usrp_source.h>
 #include <uhd/convert.hpp>
+#include <boost/thread/mutex.hpp>
```

Comment:
Is there a boost mutex in this file?

Reply:
Oh, that must have been a mistake while rebasing.

Reply:
Also fixing the other ICWYU omissions.